### PR TITLE
Align builds of the template engine used by the 1.1.x and 2.0.x CLIs

### PR DESCRIPTION
**Pull request by mlorbetske**

### CODE REVIEW

File: build/Microsoft.DotNet.Cli.DependencyVersions.props [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <CLI_NuGet_Version>4.3.0-preview4-4258</CLI_NuGet_Version>
     <CLI_WEBSDK_Version>1.0.0-alpha-20170516-2-509</CLI_WEBSDK_Version>
     <CLI_TestPlatform_Version>15.0.0</CLI_TestPlatform_Version>
-    <TemplateEngineVersion>1.0.0-beta2-20170629-269</TemplateEngineVersion>
+    <TemplateEngineVersion>1.0.0-beta2-20170710-283</TemplateEngineVersion>
     <TemplateEngineTemplateVersion>1.0.0-beta2-20170629-269</TemplateEngineTemplateVersion>
```
Comment (seancpeters) [inline build/Microsoft.DotNet.Cli.DependencyVersions.props]: We should also update the template version number.

**Comment by livarcocc** [comment]
If we make this change, we will need to get approval to do a VS insertion. Have you guys pursued that already?

**Comment by mlorbetske** [comment]
We've not pursued it - this shouldn't be a critical change, so we can likely abandon it.

@seancpeters do you agree?

**Comment by seancpeters** [comment]
Mike already closed this, but yes, I agree.

**Pull request closed without merging**